modules/product/src/main/java/com/opengamma/strata/product/rate/IborRateObservation.java

```diff
@@ -75,6 +76,15 @@ public void collectIndices(ImmutableSet.Builder<Index> builder) {
     builder.add(index);
   }
 
+  /**
```

Comment:
This method is not needed with the new `calculateFixingDateTime(LocalDate)` method on the index